### ISSUE TITLE
Migrate the project to Jakarta JAXB 3+

### ISSUE THREAD

**Issue by Spikhalskiy** [jpmml/jpmml-model#41]
When attempting to use the project in an environment with new Jakarta JAXB 3+ the following exception is thrown:

```
Caused by: java.lang.NoClassDefFoundError: javax/xml/bind/JAXBContext
	at org.jpmml.model.JAXBUtil.getContext(JAXBUtil.java:106) ~[pmml-model-1.5.15.jar:?]
	at org.jpmml.model.JAXBUtil.createUnmarshaller(JAXBUtil.java:151) ~[pmml-model-1.5.15.jar:?]
	at org.jpmml.model.JAXBUtil.unmarshal(JAXBUtil.java:56) ~[pmml-model-1.5.15.jar:?]
	at org.jpmml.model.JAXBUtil.unmarshalPMML(JAXBUtil.java:44) ~[pmml-model-1.5.15.jar:?]
	at org.jpmml.model.PMMLUtil.unmarshal(PMMLUtil.java:68) ~[pmml-model-1.5.15.jar:?]
```

Would it be possible to switch the project from `javax.*` to `jakarta.*` or to create a parallel jakarta variant of the pmml-model module utilizing eclipse-transformer?

**Comment by vruusmann** [comment]
> Would it be possible to switch the project from javax.* to jakarta.*

You are using JPMML-Model 1.5.X, which is "Java XML Bind" (aka `javax.xml.bind`)-based.

The upgrade to Jakarta XML Bind (aka `jakarta.xml.bind`) happened in late 2021, when JPMML-Model 1.6.0 was released.

Currently, the project is being migrated from Jakarta XML Bind 3 to 4 (needed for Java 8 to 11 upgrade). It will be the cornerstone of the upcoming JPMML-Model 1.7.X.

TLDR: Jakarta XML Bind 3 is in the `1.6.X` branch, and Jakarta XML Bind 4 is in the `master` branch. Both work with Glassfish Metro and EclipseLink MOXy runtimes just fine. All releases are always in the Maven Central repository.

See also https://github.com/jpmml/jpmml-evaluator/issues/246
